### ISSUE TITLE
Node.js 15 issue

### ISSUE THREAD

**Issue by ai** [shashkovdanil/clean-publish#73]
```
➜ npx clean-publish
/usr/bin/env: ‘node\r’: No such file or directory
npm ERR! code 127
npm ERR! path /home/ai/Dev/storeon
npm ERR! command failed
npm ERR! command sh -c clean-publish

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ai/.npm/_logs/2020-10-22T23_37_49_537Z-debug.log
```

@shashkovdanil maybe we use a Windows `\r\n` and it causes a problem in npm 7 on Linux?

**Comment by shashkovdanil** [comment]
Okay, I will try fix that on weekend

**Comment by ai** [comment]
Do you use `\n` or `\r\n` in your files?

**Comment by ai** [comment]
Here is the issue https://github.com/yarnpkg/yarn/issues/8106

Do you publish this npm package on Windows machine?

I checked that there is `\n` in sources.

**Comment by ai** [comment]
Fixed in 2.0
